knowledge_base/write_from_job_to_volume/src/hello.ipynb

```diff
@@ -0,0 +1,21 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "file_path = dbutils.widgets.get(\"file_path\")\n",
```

Comment:
This doesn't actually work does it? Without a `dbutils.widgets.text()` and/or `widgets` section in the ipynb JSON below

Reply:
Works fine:

<img width="674" alt="Screenshot 2024-12-09 at 10 18 18 PM" src="https://github.com/user-attachments/assets/1a246612-9177-472d-9930-50a6f9ea3f24">

```
(.venv) ➜  cli git:(detect/schema-dep) databricks fs cat dbfs:/Volumes/main/shreyas_goenka_hello_world/my_volume/hello_world.txt -p dogfood
Hello World!%
```